tests/python/PEP8NotebookBearTest.py

```diff
@@ -89,3 +119,10 @@
                       invalid_files=(bad_file[:-1],),
                       force_linebreaks=False,
                       )
+
+PEP8NotebookBearWithInfLineTest = \
```

Comment:
I would normally rant about removing `\` , but I'll leave that for another day if I see you using them again :P